web_frontend/cloudscheduler/glintwebui/glint_views.py

```diff
@@ -35,33 +36,34 @@
 LIST_KEYS = {
     # Named argument formats (anything else is a string).
     'format': {
-        'csrfmiddlewaretoken':                        'ignore',
```

Comment:
Why was this formatting change made? The whitespace existed here for readability and there is no reason to change it.